azerothcore/azerothcore-wotlk
# Fix(Core/BWL): Vaelastrasz reset after wipe

## Changes Proposed:
-  Change react state when the intro is done, as it was passive all the time so it would never had changed the initial target.
-  We shouldn't talk to him every time we wipe.
-  Separate the timers.

## Issues Addressed:

- Closes #10153

## SOURCE:


## Tests Performed:

- Tested in-game


## How to Test the Changes:


1. .go c id 13020
2.  use the gossip, wait the intro and let the tank die
3.  it should reset or change the target.

## Known Issues and TODO List:


- [x]  We shouldn't have to talk to him after every wipe. [[confirmed by [this](https://www.youtube.com/watch?v=ik6jreLykMQ) , already fixed]]
